hrui/CRM/Core/BAO/Navigation.php

```diff
@@ -675,7 +675,9 @@ public static function createNavigation($contactID) {
 
       $hideLabel = ts('Hide Menu');
       $prepandString = "
-        <li class='menumain crm-link-home'><a href='$homeURL'>$homeIcon $homeLabel</a>";
+        <li class='menumain crm-link-home'><a href='$homeURL'>
+          <span class=\"menumain-icon\">$homeIcon</span>
```

Comment:
Why do you need to use `\"` sometimes and sometimes `'`? Could we simply use `'` for JS and `"` for HTML?